lib/record_store/provider/google_cloud_dns.rb

```diff
@@ -3,7 +3,7 @@
 module RecordStore
   class Provider::GoogleCloudDNS < Provider
     class << self
-      def apply_changeset(changeset, stdout = $stdout)
+      def apply_changeset(changeset, stdout = $stdout) # rubocop:disable Lint/UnusedMethodArgument
```

Comment:
This is not an interface here, please rename the parameter `stdout` in `_stdout` (and do not set default value as we don't use it anyway) instead of ignoring the error!

I'm also confused by the interface of that class, where we sometimes have ordered parameters and sometimes named ones... But in this case we could just rename and remove default as not used (which is not possible when using named parameters as it would change the behavior)